### ISSUE TITLE
About Android and iOS support

### ISSUE THREAD

**Issue by nongvantinh** [godotengine/godot-cpp#818]
In 3.x there is Android support and docs to compile for that platform, but with the new GDExtension, is that possible? 
Compile for it still the same as before? How does it differ from branch 3.x?

But what about iOS support? Can we develop a game for iOS using GDExtension now?

**Comment by CancerQ** [comment]
You can build your own xcode project, add your source files to the project, and then rely on godot-cpp, compile it into a dynamic framework, and then directly use the dynamic library

**Comment by dsnopek** [comment]
This is a pretty old issue - I'm not sure the state of things back then, but, presently, both Android and iOS are supported. :-)

**Comment by Calinou** [comment]
> This is a pretty old issue - I'm not sure the state of things back then, but, presently, both Android and iOS are supported. :-)

Is there documentation to link to for compiling a GDExtension to Android/iOS? I'd say we need that before we can close this issue.

**Comment by dsnopek** [comment]
At the moment I don't think we have any such documentation for _any_ platform. Yesterday, someone opened issue https://github.com/godotengine/godot-cpp/issues/1282 which seems to be specifically about this

**Comment by Calinou** [comment]
Closing in favor of https://github.com/godotengine/godot-cpp/issues/1282.
